charts/prometheus/README.md

```diff
@@ -65,6 +65,11 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 20.0
+
+The [configmap-reload](https://github.com/jimmidyson/configmap-reload) container was replaced by the [prometheus-config-reloader](https://github.com/prometheus-operator/prometheus-operator/tree/main/cmd/prometheus-config-reloader).
+Extra command line arguments specified via configmapReload.prometheus.extraArgs are not compatible and will break wit the new prometheus-config-reloader, refer to the [sources](https://github.com/prometheus-operator/prometheus-operator/blob/main/cmd/prometheus-config-reloader/main.go) to make adjustments to those
```

Comment:
1. Break `wit` -> break `with`
2. `to make adjustments to those` seems incomplete.

Reply:
sorry for those mistakes... I was coming up with this PR as I was ending my workday last week....
should have waited....
trying to get this in so we can avoid pulling stuff from dockerhub (see: https://news.ycombinator.com/item?id=35154025) and I fell this project (https://github.com/jimmidyson/configmap-reload) has a risk for getting deleted or even worse (squatted).